src/bugsnag.js

```diff
@@ -790,8 +875,10 @@
   // get breadcrumb specific setting. When autoBreadcrumbs is true, all individual events are defaulted
   // to true. Otherwise they will all defaul to false. You can set any event specicically and it will override
   // the default.
-  function getBreadcrumbSetting(name) {
-    var fallback = getSetting("autoBreadcrumbs", true);
+  function getBreadcrumbSetting(name, fallback) {
```

Comment:
I’m a little confused on what’s going on here, sometimes `getBreadcrumbSetting` is called with `true` and other times with no argument. Are those resolving to the same thing? If so, I think it makes more sense to remove the `fallback` argument.